### ISSUE TITLE
Forward Perspective: Where might we be in 2029?

### ISSUE THREAD

**Issue by yochannah** [open-life-science/open-life-science.github.io#856]
The natural next step after #855. 

We'll do our own retros independently, then come together as the three of us to share our individual visions. 

At what point / how do we loop in the greater team and the community?

**Comment by yochannah** [comment]
closing this in favour of https://github.com/open-life-science/internal-policies-and-procedures/issues/222
